x-pack/plugins/cases/server/authorization/authorization.ts

```diff
@@ -59,58 +59,58 @@ export class Authorization {
-    let caseScopes: Set<string>;
+    let caseOwners: Set<string>;
     try {
       const disabledFeatures = new Set((await getSpace(request))?.disabledFeatures ?? []);
 
-      caseScopes = new Set(
+      caseOwners = new Set(
         features
           .getKibanaFeatures()
-          // get all the features' cases scopes that aren't disabled
+          // get all the features' cases owners that aren't disabled
           .filter(({ id }) => !disabledFeatures.has(id))
           .flatMap((feature) => feature.cases ?? [])
       );
     } catch (error) {
-      caseScopes = new Set<string>();
+      caseOwners = new Set<string>();
     }
 
-    return new Authorization({ request, securityAuth, caseScopes, isAuthEnabled });
+    return new Authorization({ request, securityAuth, caseOwners, isAuthEnabled });
   }
 
   private shouldCheckAuthorization(): boolean {
     return this.securityAuth?.mode?.useRbacForRequest(this.request) ?? false;
   }
 
-  public async ensureAuthorized(scope: string, operation: ReadOperations | WriteOperations) {
+  public async ensureAuthorized(owner: string, operation: ReadOperations | WriteOperations) {
     // TODO: remove
     if (!this.isAuthEnabled) {
       return;
     }
 
     const { securityAuth } = this;
-    const isScopeAvailable = this.featureCaseScopes.has(scope);
+    const isOwnerAvailable = this.featureCaseOwners.has(owner);
 
     // TODO: throw if the request is not authorized
     if (securityAuth && this.shouldCheckAuthorization()) {
       // TODO: implement ensure logic
-      const requiredPrivileges: string[] = [securityAuth.actions.cases.get(scope, operation)];
+      const requiredPrivileges: string[] = [securityAuth.actions.cases.get(owner, operation)];
 
       const checkPrivileges = securityAuth.checkPrivilegesDynamicallyWithRequest(this.request);
       const { hasAllRequested, username, privileges } = await checkPrivileges({
         kibana: requiredPrivileges,
       });
 
-      if (!isScopeAvailable) {
-        // TODO: throw if any of the scope are not available
+      if (!isOwnerAvailable) {
+        // TODO: throw if any of the owner are not available
         /**
          * Under most circumstances this would have been caught by `checkPrivileges` as
-         * a user can't have Privileges to an unknown scope, but super users
-         * don't actually get "privilege checked" so the made up scope *will* return
+         * a user can't have Privileges to an unknown owner, but super users
+         * don't actually get "privilege checked" so the made up owner *will* return
          * as Privileged.
          * This check will ensure we don't accidentally let these through
          */
         // TODO: audit log using `username`
-        throw Boom.forbidden('User does not have permissions for this scope');
+        throw Boom.forbidden('User does not have permissions for this owner');
```

Comment:
I think the user will have a better understanding of the problem, isn't it?
```suggestion
        throw Boom.forbidden(`User does not have permissions cases from ${this. featureCaseOwners}`);
```

Reply:
If you agree, I will apply that to every msgs

Reply:
Hey! This will be taken care in this PR: https://github.com/elastic/kibana/pull/95477